creativecloud/blocks/interactive-marquee/interactive-marquee.js

```diff
@@ -74,4 +74,17 @@ function interactiveInit(el) {
 
 export default async function init(el) {
   interactiveInit(el);
```

Comment:
We should move this to be called inside respective switch case as another variation of interactive marque (like chabgebg) does not want the default marquee to load and directly loads its own JS and CSS

Reply:
Sorry didn't got this part. Do we need InteractiveInit() to be called only for firefly? If yes I can move it to firefly case in switch condition

Reply:
done